templates/include/letsencrypt.conf.j2

```diff
@@ -0,0 +1,4 @@
+location ^~ /.well-known/acme-challenge/ {
+    alias {{ dehydrated.config.WELLKNOWN }}/;
```

Comment:
Should we add a remark (other than in meta) That this role depends on the dehydrated role? Documentation?

Reply:
Role dependencies are managed in `meta/main.yaml`.

Reply:
Sure, but if someone chooses to use this role, they will not necessarily be notified about the dependency. Maybe documentation is not the right place but maybe e.g. one of GitHub's mechanisms to track dependencies?

Reply:
So that it becomes clear on which exact role-project this one depends

Reply:
For new projects the [proserver-ansible-template](https://github.com/punktDe/proserver-ansible-template) repository should be used to get started. The repository contains all necessary submodules and documentation. We could also add a README to this project (some roles have this, e.g. the [PostgreSQL role](https://github.com/punktDe/proserver-ansible-postgresql)).